From 94e0df6d0f8b44228de5be0ff35efb9f946a13c9 Mon Sep 17 00:00:00 2001
From: James Zern <jzern@google.com>
Date: Wed, 2 Apr 2025 15:42:02 -0700
Subject: [PATCH] CMakeLists.txt: bump minimum cmake version to 3.5 (#132)

Fixes an error with cmake 4.0.0:
  Compatibility with CMake < 3.5 has been removed from CMake.

Also indicate that the file has been updated to support 3.10 to avoid:
  Compatibility with CMake < 3.10 will be removed from a future version
  of CMake.

3.5 is relatively old, but maintain compatibility while cmake still
supports it.

Fixes: #131
---
 .github/workflows/ci.yml | 2 +-
 CMakeLists.txt           | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d852ee2..fa848ae 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-cmake_minimum_required(VERSION 2.8.7)
+cmake_minimum_required(VERSION 3.5...3.10)
 
 # Prefer GLVND libraries by default.
 if(POLICY CMP0072)
